addons/l10n_dk/models/account_move.py

```diff
@@ -0,0 +1,21 @@
+from odoo import api, fields, models
+
+
+class AccountMove(models.Model):
+    _inherit = "account.move"
+
+    l10n_dk_currency_rate_at_transaction = fields.Float(
+        string='Rate',
+        compute='_compute_currency_rate_at_transaction', readonly=True, store=True,
+        digits=(12, 6),
```

Comment:
Is there a reason to limit the rate to 6 digits?
Would be nice to keep the same precision as in the `res.currency.rate` table
```suggestion
        digits=0,
```